### ISSUE TITLE
Logins across sub domains

### ISSUE THREAD

**Issue by sdousley** [franzliedke/auth-fluxbb#21]
Hi

Not sure if this is a bug with the Auth driver, or Laravel itself, but I'm trying to access a sub-domain (admin.domain.com) whilst logged into the main site (domain.com), however, when I hit the admin sub domain, Auth::User() returns false, not the user that is logged in on the main site.

I've got my 'domain' setting in session.php set to 'domain.com', tried with/out the leading . but neither seems to work.

Is this a potential issue within auth-fluxbb?

For a bit more info, when I view cookies in a cookie editor, I see 2 cookies.  One for .domain.com with a cookie name of "laravel_session" (what is set in my session.php) and then a seperate session for just domain.com (no leading .) with a cookie name of pun_cookie_9170ad.

This cookie name (pun_cookie_9170ad) is actually what's stored in my forum/config.php as $cookie_name.  Do i need to alter the forum cookie name or laravel cookie name, so that they both match?

Do I also need to change the cookie domain so that they both match? If so, with or without the leading full stop?

TIA.
Steve

**Comment by franzliedke** [comment]
This is likely a configuration issue.

When you visit admin.domain.com, is the "laravel_session" cookie sent along with the request? You can find out whether it is using your browser's developer tools, especially the network tab. If it is, make sure to use the same configuration for the FluxBB cookie, too. (And no, these two are fine to be two different cookies.)

**Comment by sdousley** [comment]
OK, so to run down what I have just tested

To view the cookies, I checked the resources tab in Chrome's Developer tools. All of this is done in Chrome Incognito mode.

Browse to domain.com (whilst not logged in).
Cookie Domain: domain.com
Cookie name: laravel_session
Cookie Value: eyJpdiI6IkpnUEgwemZkZFpPeXM0cTZUbE1HTEE9PSIs..........

Browse to 
Browse to admin.domain.com 
Cookie Domain: admin.domain.com
Cookie name: laravel_session
Cookie Value: eyJpdiI6IkpnUEgwemZkZFpPeXM0cTZUbE1HTEE9PSIs..........

Browse to domain.com (and log in)
Same cookie as before
but additional cookie
Cookie domain: domain.com
Cookie Name: pun_cookie_04b19e
Cookie Value: 2%7Cfded06cd1a65e9aabefe89.....

So I'm not entirely sure what's going on with this. Maybe I need to set admin/laravel to use pun_cookie_04b19e or set punBB to use laravel_session?
